GoMeet/src/main/webapp/js/dao-utils.js

```diff
@@ -0,0 +1,21 @@
+class DAOUtils {
```

Comment:
Moving this to a utils class was a smart idea :)

Reply:
Thankyou! 🤓